app/src/main/res/values/strings.xml

```diff
@@ -367,6 +367,11 @@
     <string name="symbols_numbers_keyboard">123</string>
     <string name="symbols_time_keyboard">time</string>
     <string name="eng_keyboard">English</string>
+    <string name="dvorak_keyboard">Dvorak</string>
+    <string name="compact_keyboard">Compact QWERTY</string>
+    <string name="compact_keyboard_16keys">16 Keys English</string>
```

Comment:
This should be much shorter since this text will also be shown on the keyboard's language-switch key.
Maybe "T9 English"? Or "16 Keys EN"?